### PR TITLE
Updating Treatment Code List

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Need updated list so that sample files can be validated correctly

# What has changed
Removed any treatment codes not in confluence table
Updated resource files
As treatment code quantities are TBD, have divided 350,000 by number of codes and made this number the temporary quantity for each

# How to test?
Validate a sample file and check it passes

# Links
https://trello.com/c/CRTEmBzO/1113-make-treatment-codes-correct-for-sample-validation-3
https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?spaceKey=SDC&title=Treatment+Codes+2021

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Make treatment codes correct for sample validation (3)](https://trello.com/c/CRTEmBzO/1113-make-treatment-codes-correct-for-sample-validation-3)

File: treatment_code_quantities.csv [hunk 1]
```diff
@@ -1,48 +1,34 @@
...
-HH_QFNR1W,5060,U
-HH_QFNR2E,4859,U
-HH_QFNR2W,3233,U
-HH_QFNR3AE,1426,U
-HH_QFNR3AW,3571,U
-SPG_QDHSE,747,U
-SPG_QDHSW,836,U
\ No newline at end of file
+Treatment Code,Quantity,Address Level
+HH_LP1E,10606,U
```
Comment (NickGrantX) [inline treatment_code_quantities.csv:2]: Why 10,606? Adds up to 350k I suppose?

**Review by NickGrantX — APPROVED** [review]

File: validate_sample.py [hunk 2]
```diff
@@ -10,13 +10,14 @@
...
-        'HH_QFNR1E', 'HH_QFNR2E', 'HH_QFNR3AE', 'HH_QF2R1W', 'HH_QF2R2W', 'HH_QF2R3AW', 'HH_QF3R1W', 'HH_QF3R2W',
-        'HH_QF3R3AW', 'HH_QFNR1W', 'HH_QFNR2W', 'HH_QFNR3AW', 'HH_3QSFN', 'SPG_QDHSE', 'SPG_QDHSW', 'SPG_LPHUE',
-        'SPG_QDHUE', 'SPG_VDNEE', 'CE_QDIEE', 'SPG_VDNEW', 'CE_QDIEW', 'SPG_LPHUW', 'SPG_QDHUW', 'CE_LDIUE', 'CE_LDIUW'}
+        'HH_LP1E', 'HH_LP1W', 'HH_LP2E', 'HH_LP2W', 'HH_QP3E', 'HH_QP3W', 'HH_1ALSFN', 'HH_2BLEFN',
+        'HH_2CLEFN', 'HH_3DQSFN', 'HH_3EQSFN', 'HH_3FQSFN', 'HH_3GQSFN', 'HH_4HLPCVN', 'HH_SPGLNFN', 'HH_SPGQNFN',
+        'CE_LDIEE', 'CE_LDIEW', 'CE_LDIUE', 'CE_LDIUW', 'CE_QDIEE', 'CE_QDIEW', 'CE_LDCEE', 'CE_LDCEW',
+        'CE_1QNFN', 'CE_2LNFN', 'CE_3LSNFN', 'SPG_LPHUE', 'SPG_LPHUW', 'SPG_QDHUE', 'SPG_QDHUW', 'SPG_VDNEE',
+        'SPG_VDNEW'}
+
+    # NI treatment codes do not have Q or L immediately after underscore
```
Comment (NickGrantX) [inline validate_sample.py]: I don't think this is where the comment was intended to go. It was meant to go next to the code which assumes that some treatment codes have the format `HH_Q...W` to specifically deal with Welsh questionnaires... other than that we really don't care what the treatment codes are

File: treatment_code_quantities.csv (hunk 1 above)
Comment (LeoKHoward) [reply treatment_code_quantities.csv:2]: Yes. Vicki doesnt have the actual quantities yet so just gave them all the same amount for now

**Review by NickGrantX — APPROVED** [review]
I'd get rid of the comment out of the validator altogether - it was supposed to go into Case Processor but I don't think it adds any value

File: validators.py [hunk 3]
```diff
@@ -98,6 +98,9 @@ def validate(region, **kwargs):
 
     return validate
 
+    # NI treatment codes do not have Q or L immediately after underscore
```
Comment (davidmort) [inline validators.py]: not sure this comment make sense here as it doesn't relate to the treatment code and region lining up in the sample

**Review by davidmort — CHANGES_REQUESTED** [review]
yeah that comment in validators.py does look a bit out of place

**Review by davidmort — APPROVED** [review]

**Pull request merged**